### ISSUE TITLE
Collapse buttons for launcher and workflow list not reachable by keyboard

### ISSUE THREAD

**Issue by Bubballoo3** [OSC/ondemand#5257]
It looks like the buttons to expand and collapse launcher and workflow lists inside a project need a bit of work to become accessible, as they currently 
- are not reachable via keyboard (focus skips past)
- don't respond to enter when focused
- don't contain text to explain the effect

These buttons are important because they allow users to skip a lengthy list of launchers
